src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationHandler.cs

```diff
@@ -586,34 +621,13 @@ private string ReadNonceCookie(string nonce)
             return null;
         }
 
-        private AuthenticationProperties GetPropertiesFromState(string state)
+        private AuthenticationProperties SetUserStateOnMessage(OpenIdConnectMessage message, AuthenticationProperties properties)
```

Comment:
@PinpointTownes  can you help me understand about leaving the notification null.  I thinks there is a gain here, if the user hasn't set a notification, then no need to new up the Notification and call a no-op.

+1 on removing the mini-method: SetUserStateOnMessage